mediascrapers.py

```diff
@@ -118,6 +118,24 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self._name = 'general'
 
+         if __name__ == "__main__":
```

Comment:
@sachinmavi04 this code block will not be here. 
Check the `mediascrapers` folder. Put it in a file called `twitterimagescraper.py` there. 
Refer to other files already present in `mediascrapers`.